definitions/checks/container_gateway/db_up.rb

```diff
@@ -0,0 +1,29 @@
+module Checks
+  module ContainerGateway
+    class DBUp < ForemanMaintain::Check
```

Comment:
[idea] It looks like we'll have nearly identical code at `definitions/checks/{foreman,candlepin,pulpcore,container_gateway}/db_up.rb`... what do you think about abstracting these into a common subclass of ForemanMaintain::Check?

Reply:
It sounds like a good idea to me, the files appear to be very similar.